types/recharts/index.d.ts

```diff
@@ -324,7 +325,7 @@ export interface PieProps {
 	activeShape: any | React.ReactElement<any> | RechartsFunction;
 	isAnimationActive?: boolean;
 	animationBegin?: number;
-	animationDuration?: number;
+	// animationDuration?: number;
```

Comment:
If the property is not there any more, could you please *remove* it instead of commenting it out?